e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -180,7 +180,7 @@ describe("scenarios > collection defaults", () => {
 
   it("should support markdown in collection description", () => {
     cy.request("PUT", "/api/collection/9", {
-      description: "# header",
+      description: "[link](https://metabase.com)",
```

Comment:
it's more accurate test as it's one of requirements of the tasks - to display links